push.sh

```diff
@@ -2,6 +2,13 @@
 set -euo pipefail
 source ${GITHUB_ACTION_PATH}/common.sh
 
+function fail_with_msg() {
+  # $1 = message
+  echo "$1"
```

Comment:
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#example-3

Reply:
`echo "::error::${1}"`